indico/web/assets/vars_js.py

```diff
@@ -69,6 +69,7 @@ def generate_user_file(user=None):
     else:
         user_vars = {
             'id': user.id,
+            'title': user._title.name,
```

Comment:
Can you confirm that works well even if the user has no title?

Reply:
It does, we have the same logic in `PrincipalsMixin` which is used when you search for people in the person link. But it's probably the best - we use `NoneValueEnumField` in other places, and in the frontend the `none` UserTitle is not in the options for the title dropdown either...

Maybe something like this would be better?
```py
'title': NoneValueEnumField(UserTitle, none_value=UserTitle.none).serialize('_title', user)
```
So we actually get a proper `null` like in user preferences.

(Or just using UserSchema directly but then we need to camelize the keys and handle avatarURL directly..)

Reply:
> Or just using UserSchema directly but then we need to camelize the keys and handle avatarURL directly..

you could subclass the schema and add a post_dump to camelize and handle that one separately